src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -45,7 +45,7 @@ AthensSurfaceExamples class >> draw2Strings [
 		can pathTransform translateBy: advance.
 		can drawString: 'in mind.'].
 	
-	Display getCanvas drawImage: 	surf asForm  at: 0@0
+	surf asForm asMorph openInWindow.
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?